contracts/solidity/block/BlockInfo.sol

```diff
@@ -32,7 +32,8 @@ contract BlockInfo {
     }
 
     // should behave like prevrandao
+    /// @notice since VM version Paris, "difficulty" was replaced by "prevrandao", which now returns a random number based on the beacon chain
     function getBlockDifficulty() external view returns (uint256) {
-        return block.difficulty;
+        return block.prevrandao;
```

Comment:
Q: is this a replacement due to warning?
Is there a way to ignore the warning and maintain the coverage?
Also add support for `prevrandao`

Reply:
Yes, it's because of the warning. Solidity doesn't inherently support ignoring warnings. I did some research on a third-party Hardhat plugin `hardhat-ignore-warning`, but it only works for completely turning off warnings for the entire project, which isn't a good solution. The plugin also offers a function to ignore warnings for specific lines using inline comments, but they don't seem to work. I'm reaching out to the plugin maintainer to ask about this issue. For now, we can revert back to using `block.difficulty` to maintain coverage. And we already have coverage for `prevrandao`." Pushed in the fix.